src/main/java/org/tron/common/runtime/vm/program/Program.java

```diff
@@ -1506,7 +1506,12 @@ public StackTooLargeException(String message) {
   }
 
   public DataWord getCallEnergy(OpCode op, DataWord requestedEnergy, DataWord availableEnergy) {
-    return requestedEnergy.compareTo(availableEnergy) > 0 ? availableEnergy : requestedEnergy;
+    if (requestedEnergy.compareTo(availableEnergy) > 0) {
+      return availableEnergy;
+    }
+    else {
```

Comment:
![Code Smell](https://sonarsource.github.io/sonar-github/code_smell.png 'Code Smell') Code Smell: Move this "else" on the same line that the previous closing curly brace. ([squid:RightCurlyBraceSameLineAsNextBlockCheck](https://sonarcloud.io/organizations/tron-zhaohong/rules#rule_key=squid%3ARightCurlyBraceSameLineAsNextBlockCheck))

[See it in SonarCloud](https://sonarcloud.io/project/issues?id=java-tron&pullRequest=1633&issues=AWalyDYZdPPRVdwylpM4&open=AWalyDYZdPPRVdwylpM4)

[ISSUE_KEY]: AWalyDYZdPPRVdwylpM4